unity-environment/Assets/ML-Agents/Editor/AgentEditor.cs

```diff
@@ -59,7 +59,7 @@ public override void OnInspectorGUI()
         EditorGUILayout.PropertyField(
             isODD,
             new GUIContent(
-                "On Demand Decision",
+                "On Demand Decisions",
```

Comment:
Can you update the images on the doc as well ?

Reply:
That will be a separate PR when I update those pages ... made a note.